frontend/components/GameSpoilersList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
+import {AppStore} from '../storage/common';
+// eslint-disable-next-line no-unused-vars
+import { GameSpoilers } from '../storage/models/GameSpoilers';
+interface GameSpoilersListProps{
+	gameSpoilers: Map<number, GameSpoilers>;
+}
+const GameSpoilersList: React.FC<GameSpoilersListProps> = (props: GameSpoilersListProps) =>{
+//I couldn't do destructuring, because i get following error: 'Property 'gameSpoilers' does not exist on type 'Map<number, GameSpoilers>''
+	const gameSpoilers = props.gameSpoilers;
+
+	return (
+		<React.Fragment>
+			{[...gameSpoilers.values()].map(game=> {
```

Comment:
I personally prefer `Array.from()` rather than `[...x]` when converting something that is not array to array. It's clear from the language that you want to get an array, while spread operator has many more applications. It's not a hill I am willing to die on, if you'd like to play more with spread operator by all means keep it as-is - just apply it consistently and I'll be content :).

Reply:
Yeah, Array from is much more readable. TBH when I first found example of iterating through maps I had to think what that part is exactly :)